src/provider/model/Model.ts

```diff
@@ -31,6 +31,11 @@ interface ExpectedWindow {
     closed: Promise<void>;
 }
 
+interface WindowGroup {
```

Comment:
Not sure I like the naming here. There is already an established concept of a `WindowGroup` in OpenFin (not that we use it in FDC3, but still). Really this is just a single object containing an app and it's main/child windows, so couldn't we just have the windows array as a property of the application. I know that's probably a lot more work than this, but makes more sense to me in terms of the hierarchy given that an App is really just a collection of windows plus some other state.

Reply:
I don't think that would be a property of the `Application`, I'd want to keep that as a representation of the directory data. A new object to represent a running app could make sense, but that would mean big changes to `Model` and `Environment`, which feels beyond this PR. Open to push back on that though

Reply:
Renamed `WindowGroup` to `LiveApp`, which I think is an improvement, and consistent with terminology I've used elsewhere

Reply:
Still think that the distinction would make sense, but happy with the name change for now.